src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/XamlRequestExecutionQueue.cs

```diff
@@ -23,15 +24,15 @@ internal class XamlRequestExecutionQueue : RequestExecutionQueue<RequestContext>
             _projectService = projectService;
         }
 
-        protected override string GetLanguageForRequest<TRequest>(string methodName, TRequest request)
-        {
-            if (request is ITextDocumentParams textDocumentParams &&
-                textDocumentParams.TextDocument is { Uri: { IsAbsoluteUri: true } documentUri })
-            {
-                _projectService.TrackOpenDocument(documentUri.LocalPath);
-            }
+        //internal override string GetLanguageForRequest(string methodName, JObject request)
+        //{
+        //    if (request is ITextDocumentParams textDocumentParams &&
+        //        textDocumentParams.TextDocument is { Uri: { IsAbsoluteUri: true } documentUri })
+        //    {
+        //        _projectService.TrackOpenDocument(documentUri.LocalPath);
```

Comment:
Ah yes, this is for the VS in-proc LSP server. It would probably break the internal preview for it, although I haven't tried the VS scenario in a long time.

Reply:
@mgoertz-msft how important is this to keep working?  Likely would need to add another virtual to the queue to keep this around.

Is this a live scenario, or will this be entirely replaced by XAML lsp support using language handlers?